extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -0,0 +1,47 @@
+define('AuthModal', ['jquery', 'AuthComponent', 'wikia.window'], function ($, AuthComponent, window) {
+	'use strict';
+
+	var modal,
+		$blackout,
+		language = window.wgContentLanguage;
```

Comment:
IMO, UserLanguage is more suitable here, eg. user types ```uselang=es``` on EN page, UI language on auth pages should be consistent with language on current page.